specs/AppLifecycle/Activation/AppLifecycle Activation.md

```diff
@@ -344,23 +352,35 @@ Similarly, if the app registers multiple times for startup, this is also
+| static void RegisterForProtocolActivation(<br/> string name,<br/> string exePath<br/>); | **name**:<br/>the protocol identifier, eg "foo" or "https".<br/> **exePath**: path to the<br/>executable to be activated. |
+| static void RegisterForProtocolActivation(<br/> string name<br/> ); |                                                              |
+| static void RegisterForStartupActivation(<br/><br/> string taskId,<br/> bool isEnabled,<br/> string displayName,<br/> string exePath); | **taskId**: a string identifier which the app can use in<br/>the StartupTask API to request enabling or disabling of the startup<br/>behavior.<br/> **isEnabled**: enables the startup behavior without a user prompt,<br/>so long as the user has activated the app at least once. Consistent with<br/>existing Desktop Bridge behavior.<br/> **displayName**: used in the Settings -<br/>Apps - Startup page.<br/> **exePath**: path to the executable to be activated. |
+| RegisterForStartupActivation(<br/> string taskId,<br/> bool isEnabled,<br/> string displayName<br/> ); |                                                              |
+
+
 
 ### Activation unregistration
 
 The new API will include UnregisterForXXXActivation methods for each of the
 supported activation kinds.
 
-| API                                                                        | Description                                                                   |
-| -------------------------------------------------------------------------- | ----------------------------------------------------------------------------- |
-| static void UnregisterForFileTypeActivation(<br> string fileTypes[]<br> ); | **fileTypes**: one or more filetypes that the caller wants to unregister.     |
-| static void UnregisterForProtocolActivation(<br> string name<br> );        | **name**: the previously-registered protocol identifier, eg "foo" or "https". |
-| static void UnregisterForStartupActivation();                              |                                                                               |
+| API                                                          | Description                                                  |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| static void UnregisterForFileTypeActivation(<br> string fileTypes[], <br> string exePath<br> ); | **fileTypes**: one or more filetypes that the caller wants to unregister. <br>**exePath**: path to the executable that was registered for activation. |
+| static void UnregisterForFileTypeActivation(<br> string fileTypes[]<br> ); |                                                              |
+| static void UnregisterForProtocolActivation(<br> string name,<br> string exePath<br> ); | **name**: the previously-registered protocol identifier, eg "foo" or "https". <br>**exePath**: path to the executable that was registered for activation. |
```

Comment:
Why is there no logo support for Protocol?

Reply:
Fixed.